### PR TITLE
Add secretless option for new developers

### PR DESCRIPTION
**Pull request by davidmfinol**
With this change, new developers should be able to simply clone the repo and run it with `flutter run -d chrome -t lib/main_web.dart --dart-define=SDD_SECRETLESS=true`

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -27,6 +31,6 @@ Future<Iterable<int>> getPreBuiltDeckCardIds(PreBuiltDeck deck) async {
 
 You should now be able to build and run the app.
 
-If you add a new route, set up a class for dependency injection, create a sealed class, ... you'll need to initialise the code generation.
+If you add a new route, set up a class for dependency injection, create a sealed class, ... you'll need to initialize the code generation.
```
Comment (BramDC3) [inline docs/getting_started.md:34]: No need to change this to American English :)

File: lib/packages/core/core_config/core_config_interface/lib/src/models/app_config.dart [hunk 2]
```diff
@@ -11,7 +11,8 @@ class AppConfig {
 
   String get twitterUrl => 'https://twitter.com/CrownCorpBE';
   String get tweetUrl => 'https://twitter.com/{0}/status/{1}';
-  String get youtubeUrl => 'https://www.youtube.com/channel/UCz2Que3btHnGkwrW-emZV5A';
+  String get youtubeUrl =>
+      'https://www.youtube.com/channel/UCz2Que3btHnGkwrW-emZV5A';
```
Comment (BramDC3) [inline lib/packages/core/core_config/core_config_interface/lib/src/models/app_config.dart:15]: Line length can be 120 characters

**Comment by BramDC3** [comment]
I wonder if it would be better to create a fallback for when there are no secrets instead of creating a new flavour

**Comment by davidmfinol** [comment]
> Line length can be 120 characters

I had to change `dart.lineLength` in my VSC's `settings.json` from 80 to 120. If you're using Visual Studio Code, are there any other settings I should change? Should I turn off `editor.formatOnType`/`editor.formatOnSave`? If you're using another IDE, which one?

> I wonder if it would be better to create a fallback for when there are no secrets instead of creating a new flavour

Maybe? A fallback would have the advantage of having a simpler run command/ even less complexity for new joiners.
Could it introduce the possibility of making a production build that accidently doesn't contain the secrets?
I'm not yet familiar enough with Flutter to help with such a fallback.

**Comment by BramDC3** [comment]
I think you should be good with the line length, the latter seems to be a personal preference.

I think having a fallback would reduce the complexity, and we always have a beta test before we push the apps to production so I don't think we need to worry about that.

**Comment by davidmfinol** [comment]
I'll try looking into having a fallback for the secrets instead of this flavor.
No promises, but I'll see what I can do this weekend.

**Pull request closed without merging**